### DISCUSSION TITLE
newt new myproj: pathspec 'mynewt_1_10_0_tag' did not match any file(s) known to git

### DISCUSSION THREAD

**Discussion by rtek1000** [hathach/tinyusb#1824] [Q&A]
Hi,

I'm trying to use mynewt with STM32F4, but I can't start the first project with mynewt, apparently there's a problem with git, but I couldn't solve it with the tips I found on the internet.

> ~/dev$ newt new myproj
> Downloading project skeleton from apache/mynewt-blinky...
> Downloading repository mynewt-blinky (commit: master) from https://github.com/apache/mynewt-blinky.git
> WARNING: Creating a shallow clone by default. To unshallow repository later use "git fetch --unshallow".
> Error: error: pathspec 'mynewt_1_10_0_tag' did not match any file(s) known to git

[OK] https://mynewt.apache.org/latest/newt/install/newt_linux.html

[Fail] https://mynewt.apache.org/latest/get_started/project_create.html

https://stackoverflow.com/questions/5989592/git-cannot-checkout-branch-error-pathspec-did-not-match-any-files-kn

**Comment by hathach** [comment]
I forgot the detail, but you can follow example here https://github.com/hathach/mynewt-tinyusb-example and its build script for ci https://github.com/hathach/mynewt-tinyusb-example/blob/master/.github/workflows/build.yml
